java/src/main/java/ai/rapids/cudf/DType.java

```diff
@@ -78,6 +78,10 @@ public int getNativeId() {
     return nativeId;
   }
 
+  public int getSizeInBytes() {
```

Comment:
Can we add some docs to this?  Especially explaining that this only works for fixed width types.  Variable width types like strings the value is undefined and should be ignored.

Reply:
thx ,Done